src/components/UserActions/UserActions.ts

```diff
@@ -0,0 +1,231 @@
+import { Component, IComponentBindings, Initialization, ComponentOptions, QueryEvents, l } from 'coveo-search-ui';
+
+import { InitializationUtils } from '../../utils/initialization';
+import { ResponsiveUserActions } from './ResponsiveUserActions';
+import { arrowDown } from '../../utils/icons';
+import './Strings';
+import { ClickedDocumentsList } from './ClickedDocumentsList';
+import { QueriesList } from './QueriesList';
+import { UserActivity } from './UserActivity';
+
+/**
+ * Initialization options of the **UserActions** class.
+ */
+export interface IUserActionsOptions {
+    /**
+     * Identifier of the user from which Clicked Documents are shown.
+     *
+     * **Require**
+     */
+    userId: string;
+
+    /**
+     * Label of the button used to open the user actions.
+     *
+     * Default: `User Actions`
+     */
+    buttonLabel: string;
+
+    /**
+     * Label of the summary section.
+     *
+     * Default: `Session Summary`
+     */
+    summaryLabel: string;
+
+    /**
+     * Label of the activity section.
+     *
+     * Default: `User Recent Activity`
+     */
+    activityLabel: string;
+}
+
+/**
+ * Display a panel that contains a summary of a user session and that also contains detailed information about user actions.
+ */
+export class UserActions extends Component {
+    /**
+     * Identifier of the Search-UI component.
+     */
+    static ID = 'UserActions';
+
+    /**
+     * Default initialization options of the **UserActions** class.
+     */
+    static options: IUserActionsOptions = {
+        userId: ComponentOptions.buildStringOption({ required: true }),
+        buttonLabel: ComponentOptions.buildStringOption({
+            defaultValue: 'User Actions'
+        }),
+        summaryLabel: ComponentOptions.buildStringOption({
+            defaultValue: 'Session Summary'
+        }),
+        activityLabel: ComponentOptions.buildStringOption({
+            defaultValue: 'User Recent Activity'
```

Comment:
@stremblay2

Reply:
I would go with "User's Recent Activity" in that case (yessss)